nsca_client/models/nsca_server.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2015 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
```

Comment:
@elicoidal I know it is in the guidelines but honestly, this kind of review is less useful just like the guys who went full on pep8 before it was covered by CI.